OPTIONS.md

```diff
@@ -0,0 +1,46 @@
+# VarTable Options
+
+## (Required)
+**--input-vcf**		input VCF file
+
+*	Do we need to differentiate between NGS_Mapper VCF (every position) versus lofreq VCF (only variants)?*
```

Comment:
Right now we need know that ahead of time, but we could actually auto-detect. 
Currently we use the flag `--type ( base_caller | lofreq )`

Reply:
Good stuff